### PR TITLE
Test Pull Request

### PR DESCRIPTION
**Pull request by player-two**
This is just a test to be able to show the comments and diffs in neovim.

### CODE REVIEW

**Comment by player-two** [comment]
some generic comment
other line

File: src/Main.purs [hunk 1]
```diff
@@ -28,7 +28,7 @@ main = command "CritiqPR" defaultOpts handle
 handle :: forall e. Vim -> Args -> Range -> Aff (buffer :: BUFFER, cp :: CHILD_PROCESS, fs :: FS, http :: HTTP, os :: OS, plugin :: PLUGIN | e) Unit
 handle vim args _ = case uncons args of
                          Just { head: num } -> case fromString num of
-                                                    Just x -> pull vim x
+                                                    Just b -> pull vim b
```
Comment (player-two) [inline src/Main.purs:31]: this is bad
you should feel bad

Comment (player-two) [reply src/Main.purs:31]: man that was savage

File: src/Main.purs [hunk 2]
```diff
@@ -28,7 +28,7 @@ main = command "CritiqPR" defaultOpts handle
...
                                                     Nothing -> reportError vim (num <> " is not a number")
                          Nothing -> reportError vim "PR number argument required"
```
Comment (megalord) [inline src/Main.purs]: single comment

**Pull request closed without merging**